### PR TITLE
(EWL-902): create ErrorBanner element ❗️

### PR DESCRIPTION
**Pull request by JulienKode**
# [(EWL-902)](https://eurosportdigital.atlassian.net/browse/EWL-902): create ErrorBanner element ❗️

## Checklist

- Browser compatibility checked:
  - [x] Safari
  - [x] Chrome
  - [x] Firefox
- [x] MR title respects the format: `(EWL-XXX): description`
- [x] Move Jira ticket into PR

### Error Banner

|         Expectation         |      Reality     |  
| :--------------------: | :--------------------: | 
| Screenshot 2019-07-16 at 18 36 03 | Screenshot 2019-07-16 at 18 41 03 | 

## Description

you can test the ErrorBanner here: https://deploy-preview-255--es-web-toolkit.netlify.com

- [x] create `ErrorBanner` element 
- [x] add `error-icon` to assets

## Test Plan

- create an default ErrorBanner component in stories

And then I've tested it with different size of screens

### CODE REVIEW

File: src/assets/error-icon.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                    </mask>
+                    <use id="Combined-Shape" fill="#979797" fill-rule="nonzero" xlink:href="#path-1"></use>
+                    <g id="↳-Change-colour" mask="url(#mask-2)" fill="#FF0043">
+                        <rect id="ActionTwo-Dark-Base" x="0" y="0" width="32" height="32"></rect>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (david-sautet) [inline src/assets/error-icon.svg]: I think you can minify your svg to remove unused tags

Comment (JulienKode) [reply src/assets/error-icon.svg]: @david-sautet this should solve 3a106504fff1619d0bf44d1ada4d472340b2099d the issue

**Comment by JulienKode** [comment]
@david-sautet Thanks for your review 😊 I've made an update based on your review

File: src/assets/error-icon.svg (hunk 1 above)
Comment (JulienKode) [reply src/assets/error-icon.svg]: I did the same for aaab843886db4b005c2c1d321add57ea920b2b1a in #253

**Pull request merged**